sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -3901,35 +3911,32 @@ piEnqueueMemBufferMap(pi_queue Queue, pi_mem Buffer, pi_bool BlockingMap,
   ZE_CALL(zeCommandListAppendWaitOnEvents(ZeCommandList, NumEventsInWaitList,
                                           ZeEventWaitList));
 
-  // TODO: Level Zero is missing the memory "mapping" capabilities, so we are
```

Comment:
Instead of just removing many comments, please adjust them with the current state of the matter.

Reply:
OK